Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM openjdk:8u131-jre-alpine
+FROM adoptopenjdk/openjdk8:jre8u212-b03
```

Comment:
So, not Alpine anymore?

Reply:
I don't really trust the Alpine Linux project 😬 

Anyway I can put the Alpine image if you prefer.

Reply:
I just wanted to make sure this was intentional. 😅